topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: tutorial_hands_on
+subtopic: deconvo
+priority: 3
+
+title: Creating the bulk RNA-seq dataset for deconvolution
+zenodo_link: ''
```

Comment:
The zenodo link is missing here, so links in the import data section are invalid.

Reply:
thanks, doh, should've caught that, sorry!